methods.js

```diff
@@ -26,13 +25,14 @@ module.exports = {
   },
   position: {
     async: {
-      buildChooseOne: require('./position/async/buildChooseOne'),
+      // buildChooseOne: require('./position/async/buildChooseOne'),
+      // buildMeetingTime: require('./position/async/buildMeetingTime'),
+      // buildPosition: require('./position/async/buildPosition'),
       publishChooseOne: require('./position/async/publishChooseOne'),
-      buildPosition: require('./position/async/buildPosition'),
-      publishPosition: require('./position/async/publishPosition')
+      publishMeetingTime: require('./position/async/publishMeetingTime')
     },
     sync: {
-      isChooseOnePosition: () => isChooseOnePosition
+      isPosition: () => isPosition
```

Comment:
since methods are attached to `isPosition` like `isPosition.chooseOne` opted to simplify this

for other methods in the external API, I chose to disable everything I didn't think users needed